build.gradle

```diff
@@ -283,7 +278,6 @@ project(':core') {
     testCompile project(':clients').sourceSets.test.output
     testCompile 'org.apache.hadoop:hadoop-minikdc:2.7.1'
     testRuntime "$slf4jlog4j"
-    zinc 'com.typesafe.zinc:zinc:0.3.7'
```

Comment:
Right, I didn't see any good reason to explicitly define it anymore. It used to be required for incremental builds, but now gradle is smart enough to handle it.